packages/ffe-lists-react/src/DescriptionList.js

```diff
@@ -23,6 +24,8 @@ DescriptionList.propTypes = {
     medium: bool,
     /** Large list. Can't be used with `medium` prop */
     large: bool,
+    /** Displays dts and dds inline*/
+    horizontal: bool,
```

Comment:
Please add this to the index.d.ts file as well (with a ?, since it's optional)

Reply:
https://github.com/SpareBank1/designsystem/pull/632/commits/edca781aeb0019122ca5c1bdb53f41a97c0d1c3e